app/pulp/app/tests/models/test_content.py

```diff
@@ -94,15 +94,6 @@ def test_reading(self):
                 self.assertEqual(artifact.file.read(), fp.read())
             artifact.file.close()
 
-    def test_publishing_repository(self):
-        self.associate()
-        self.add_artifacts()
-
-        # publishing
-        for content in (c.cast() for c in self.repository.content.filter(type=TYPE)):
-            for artifact in content.artifacts.all():
-                self.publish(artifact)
-
```

Comment:
Unrelated to this PR, but did we ever open an issue to tackle the problem of examples versus tests to ensure that example code is also run?